File: front/_models/university.ts
export default class University {
    university_id: number;
    name: string;
    img_file: string;

    constructor(universiti_id: number, name: string, img_file: string = "") {
        this.university_id = universiti_id;
        this.name = name;
        this.img_file = img_file;
    }
}

File: back-csharp/_data/University.cs
using System;
using System.Collections.Generic;

namespace back_csharp._data
{
    public partial class University
    {
        public University()
        {
            Campuses = new HashSet<Campus>();
        }

        public int UniversityId { get; set; }
        public string Name { get; set; } = null!;
        public DateTime CreatedAt { get; set; }
        public DateTime? ModifiedAt { get; set; }

        public virtual ICollection<Campus> Campuses { get; set; }
    }
}
